boards/px4/fmu-v5/nuttx-config/nsh/defconfig

```diff
@@ -200,7 +200,9 @@ CONFIG_STM32F7_USART2=y
 CONFIG_STM32F7_USART3=y
 CONFIG_STM32F7_USART6=y
 CONFIG_STM32F7_USART_BREAKS=y
+CONFIG_STM32F7_USART_INVERT=y
```

Comment:
Was this done for FMUv5X?

Reply:
Not specifically. Any other driver could use it, for example frsky_telemetry.